### PR TITLE
[Level 6 and cleanup] Migrate radio schedule to emotion

### PR DESCRIPTION
**Pull request by ryanmccombe**
Resolves #NUMBER

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-radio-schedule/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.6 | [PR#?](https://github.com/bbc/psammead/pull/?) Migrate to emotion. |
```
Comment (RichardPK) [inline packages/components/psammead-radio-schedule/CHANGELOG.md]:
```suggestion
| 4.0.6 | [PR#3921](https://github.com/bbc/psammead/pull/3921) Migrate to emotion. |
```

**Review by RichardPK — APPROVED** [review]

File: packages/components/psammead-radio-schedule/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.6 | [PR#3921](https://github.com/bbc/psammead/pull/3921) Migrate to emotion. |
```
Comment (simonsinclair) [inline packages/components/psammead-radio-schedule/CHANGELOG.md]:
```suggestion
| 5.0.0 | [PR#3921](https://github.com/bbc/psammead/pull/3921) Migrate to emotion. |
```

File: packages/components/psammead-radio-schedule/package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-radio-schedule",
-  "version": "4.0.5",
+  "version": "4.0.6",
```
Comment (simonsinclair) [inline packages/components/psammead-radio-schedule/package-lock.json]:
```suggestion
  "version": "5.0.0",
```

File: packages/components/psammead-radio-schedule/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-radio-schedule",
-  "version": "4.0.5",
+  "version": "4.0.6",
```
Comment (simonsinclair) [inline packages/components/psammead-radio-schedule/package.json]:
```suggestion
  "version": "5.0.0",
```

**Review by simonsinclair — APPROVED** [review]

**Comment by RichardPK** [comment]
giphy

**Pull request merged**